install.xml

```diff
@@ -34,7 +35,7 @@
 	</phrases>
 
 	<blocks>
-		<block title="Our clients" name="clients_on_map" position="bottom" type="smarty"  classname="box-clear" filename="block.clients_on_map"><![CDATA[]]></block>
+		<block title="Our clients" name="clients_on_map" position="bottom" type="smarty"  classname="box-clear" filename="block.clients_on_map"><[CDATA[]]></block>
```

Comment:
Thanks, yes it was a typo and I did create another fix but with merging and stuff it seems to have slipped somewhere or is still waiting for me to do something?